[DOCS] Add a Spark DataFrame example (#2204)
great-expectations/great_expectations: docs/guides/tutorials/explore_expectations_in_a_notebook.rst

        )

    This method will convert your boring old pandas ``DataFrame`` into a new and exciting great_expectations ``PandasDataset``. The two classes are absolutely identical, except that ``PandasDataset`` has access to Great Expectations' methods.
    
    - a ``Spark DataFrame``, you can use ``SparkDFDataset``:
    
    .. code-block:: python
    
        from great_expectations.dataset.sparkdf_dataset import SparkDFDataset
    
        my_df = SparkDFDataset(my_spark_dataframe)
    

>     This method will convert your ``Spark DataFrame`` into a class with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.

Reviewer: ```suggestion
    This method will create an object with access to Great Expectations' methods, such as ``ProfilingResultsPageRenderer``.
```

Reviewer: The "convert" language is correct for Pandas, but not precise for Spark DataFrames.
